### PR TITLE
feat(pagination): add `scrollTo` option

### LINKED ISSUE #73: API proposal: pagination widget should scroll to top by default
In most situations you want to scroll to top when a pagination element is clicked.

To disable it:
```js
search.addWidget(
  instantsearch.widgets.pagination({
    container: '#pagination',
    cssClass: 'pagination',
    maxPages: 20,
    scrollTo: '#top'
  })
);
```

We will later on need to be able to get an event when a pagination element is clicked so that the user can scroll to a specific element (like no to top but a little below the header).

For now it's not necessary to add this.

### PR DESCRIPTION
**Pull request by vvo**
Allows to scroll to a particular element/CSSSelector after a click.

On by default (scroll to body).

Can be deactivated by setting it to false.

fixes #73

### CODE REVIEW

**Comment by redox** [comment]
:+1: LGTM, didn't know about `scrollIntoView` btw :)

File: widgets/pagination.js [hunk 1]
```diff
@@ -24,9 +25,11 @@ function pagination({
     labels,
     maxPages,
     showFirstLast,
-    hideWhenNoResults = true
+    hideWhenNoResults = true,
+    scrollTo = 'body'
   }) {
   var containerNode = utils.getContainerNode(container);
+  var scrollToNode = scrollTo !== false ? utils.getContainerNode(scrollTo) : false;
```
Comment (pixelastic) [inline widgets/pagination.js]: This will fail with `scrollTo: true`

Comment (vvo) [inline widgets/pagination.js]: Yes, you will get "Container must be string or .." right?

Comment (vvo) [inline widgets/pagination.js]: Do you mean you would like "scrollTo: true" to behave like scrollTo: 'body'?

Comment (pixelastic) [inline widgets/pagination.js]: The documentation suggest that I could use a boolean here. So yes, I would make `true` the same as the default.

Comment (vvo) [inline widgets/pagination.js]: done

**Comment by pixelastic** [comment]
LGTM, merging

**Pull request merged**